XamlControlsGallery/XamlControlsGallery.csproj

```diff
@@ -1050,7 +1058,7 @@
       <Version>6.1.9</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.UI.Xaml">
-      <Version>2.0.181011001</Version>
+      <Version>2.1.181217001-prerelease</Version>
```

Comment:
Is this version of the NuGet package posted to the public feed yet?

Reply:
[Yep](https://www.nuget.org/packages/Microsoft.UI.Xaml/absoluteLatest)

Reply:
Awesome. In this case, I think we should separate this into two changes - one that bumps up the SDK and NuGet package, and one that adds the ItemsRepeater stuff. I'll bump the SDK/NuGet package reference first and then take this PR.